core/src/main/java/ai/timefold/solver/core/impl/localsearch/decider/acceptor/tabu/MoveTabuAcceptor.java

```diff
@@ -3,22 +3,16 @@
 import java.util.Collection;
 import java.util.Collections;
 
-import ai.timefold.solver.core.impl.heuristic.move.Move;
+import ai.timefold.solver.core.api.move.Move;
 import ai.timefold.solver.core.impl.localsearch.scope.LocalSearchMoveScope;
 import ai.timefold.solver.core.impl.localsearch.scope.LocalSearchStepScope;
 
 public class MoveTabuAcceptor<Solution_> extends AbstractTabuAcceptor<Solution_> {
 
-    protected boolean useUndoMoveAsTabuMove = true;
```

Comment:
Will removing the TS undo moves affect any existing code?

Reply:
Yes and no. The undo tabu no longer does anything - so in that sense, the user code will be affected. But all the public APIs are unaffected - so user code will still compile and run without any issues.

Considering I didn't even _know_ we had undo move tabu before I started looking into Move Streams, I think it's safe to say that it's an obscure feature which won't be missed.